src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Init/DirectDebitResponseSaver.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Payment\Handler\Saver\Init;
+
+use Generated\Shared\Transfer\ComputopDirectDebitInitResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+
+class DirectDebitResponseSaver extends AbstractResponseSaver
```

Comment:
Same comments as for EasyCreditResponseSaver

Reply:
Done.